### PR TITLE
V3 voices

### PR DESCRIPTION
**Pull request by leonrch**

### CODE REVIEW

**Comment by repjarms** [comment]
@leonrch the build is failing due to linter errors being caught by travis. Can you please fix these in a commit and add it to this PR?

```
/home/travis/build/watson-developer-cloud/text-to-speech-nodejs/voices.js
  306:1  error  Expected indentation of 2 spaces but found 3               indent
  435:1  error  Expected indentation of 2 spaces but found 3               indent
  486:7  error  'sortedByName' is assigned a value but never used          no-unused-vars
  508:1  error  Too many blank lines at the end of file. Max of 0 allowed  no-multiple-empty-lines
```

File: voices.js [hunk 1]
```diff
@@ -61,6 +77,22 @@ const voices = [
       ssml: JP_SSML,
     },
     description: 'Emi: Japanese (日本語) female voice.',
+  // }, {
```
Comment (repjarms) [inline voices.js]: Please delete any commented out code.

**Comment by repjarms** [comment]
Checks have passed. I'll test it out and then update my review

**Comment by germanattanasio** [comment]
@leonrch you need to configure your github locally to match your public account otherwise github doesn't recognize who made the commits.

**Comment by germanattanasio** [comment]
and we should squash this PR

**Comment by repjarms** [comment]
@german agreed on squashing

**Comment by repjarms** [comment]
@leonrch you may need to close this PR and re-create it to remove the history from the PR itself.

**Pull request closed without merging**